src/v2/guide/syntax.md

```diff
@@ -80,6 +80,8 @@ These expressions will be evaluated as JavaScript in the data scope of the owner
 
 <p class="tip">Template expressions are sandboxed and only have access to a whitelist of globals such as `Math` and `Date`. You should not attempt to access user defined globals in template expressions.</p>
 
+<p class="tip">ESNext features are available in template expressions with the caveat that the feature set is limited by the **Node.js** version you use.</p>
```

Comment:
A minor nitpick, but this supposes that rendering occurs on the server, not on the client.